app/src/main/java/tech/redroma/yoching/activities/YoActivity.kt

```diff
@@ -55,29 +55,6 @@ class YoActivity : AppCompatActivity(), NavigationMenuListener
         super.onAttachFragment(fragment)
     }
 
-    override fun onCreateView(name: String?, context: Context?, attrs: AttributeSet?): View?
```

Comment:
This would have taken me forever to figure out!

Ugh!!
![bitmoji](https://render.bitstrips.com/v2/cpanel/9945113-288439948_4-s1-v1.png?transparent=1&palette=1&width=246)